### ISSUE TITLE
Lernen - bestimmte Dinge zurücksetzen

### ISSUE THREAD

**Issue by ghost** [cwansart/voc_trainer#24]
Anfangs müssen bestimmte Dinge zurückgesetzt werden, vor allem wenn man die seite verlässt ohne die Kartei zu Ende gelernt zu haben. Das läuft noch nicht so.
Auch das leeren der textfelder muss noch übernommen werden.
Das könnten wir uns auch mal zusammen anschauen, da hast du dich ja etwas mehr mit beschäftigt die letzten tage? :)

**Comment by apollo7321** [comment]
Klar, können wir machen. Da sind bestimmte Events bei denen wir einfach alles leeren und zurücksetzen müssen.

**Comment by apollo7321** [comment]
Ist das nun auch fertig?
